terraform/src/facility.tf

```diff
@@ -80,6 +88,16 @@ resource "aws_api_gateway_integration" "writeFacilityIntegration" {
   uri                     = aws_lambda_function.writeFacilityLambda.invoke_arn
 }
 
+resource "aws_api_gateway_integration" "putIntegration" {
```

Comment:
Need to change `putIntegration` to something else since it'll conflict with the one in main.tf.  Probably something like `putFacilityIntegration`

Reply:
addressed in d225a97